packages/enclave-dashboard/src/styles.css

```
@@ -0,0 +1,2089 @@
+  color: var(--ink-4);
+  font-weight: 500;
+  letter-spacing: 0.01em;
+  padding-top: 1px;
+  white-space: nowrap;
+}
+.dl dd {
+  margin: 0;
+  font-size: 13.5px;
+  color: var(--ink);
+  min-width: 0;
+  word-break: break-word;
```

Stylelint (17.11.1)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)